changelog.mdx

```diff
@@ -5,6 +5,38 @@ description: Updates to Chalk!
 
 ---
 
+
+## August 5, 2024
+
+* We now support [DynamoDB](/docs/dynamodb) as a data source! After connecting your AWS credentials, Chalk 
+  automatically supports connecting to your DynamoDB instance. Your queries will be run with
```

Comment:
maybe "...connecting to your DynamoDB instance, and you can query them with [PartiQL](...)"? Run with partiql makes partiql seem like some sort of compiler or execution engine or something else